tools/docker/syzbot/Dockerfile

```diff
@@ -68,6 +68,11 @@ RUN test "$(uname -m)" != x86_64 && exit 0 || \
     qemu-utils qemu-system-misc qemu-system-x86 qemu-system-arm qemu-system-aarch64  \
     qemu-system-s390x qemu-system-mips qemu-system-ppc
 
+# Install ARM QEMU packages on ARM64 machines.
+RUN test "$(uname -m)" != aarch64 && exit 0 || \
+    DEBIAN_FRONTEND=noninteractive apt-get install -t bookworm-backports -y -q \
```

Comment:
These overlap with those only for `x86_64`. Maybe split that comand instead? One installs qemu for all arches (we aim only at x86 and arm64 anyway) and the other, smaller, command only those that are for `x86_64`.

Reply:
Makes sense, will do!

Reply:
PTAL